package.json

```diff
@@ -5,11 +5,8 @@
   },
   "description": "Strongly typed Cap'n Proto implementation for the browser and Node.js using TypeScript",
   "devDependencies": {
-    "@types/benchmark": "1.0.30",
-    "@types/debug": "0.0.29",
-    "@types/node": "^8.0.11",
-    "@types/ramda": "^0.24.3",
     "codecov": "^2.2.0",
+    "glob": "^7.1.2",
```

Comment:
you added this to dependencies but I think its just a devDependency, but let me know if i was mistaken.